src/main.py

```diff
@@ -198,12 +204,14 @@ def build(self):
 
         # Initialise settings object
         sett = settings_manager.Settings(sm)
```

Comment:
Why initialise "sett" at all? just use self.settings_manager. In the long run, objects should get their things through app.get_running_app anyways.